test/test-functions

```diff
@@ -778,9 +778,23 @@ if [[ ! -e "$ASAN_RT_PATH" ]]; then
     exit 1
 fi
 
+# Suppress certain leaks reported by LSan (either in external tools or bogus
+# ones)
+# Docs: # https://github.com/google/sanitizers/wiki/AddressSanitizerLeakSanitizer#suppressions
+#
+# - fsck is called by systemd-homed and is reporting a leak we're not interested
+#   in
+# - libLLVM is a "side effect" caused by the previous fsck leak
+cat >/systemd-lsan.supp <<INNER_EOF
+leak:/bin/fsck$
+leak:/sbin/fsck$
+leak:/lib/libLLVM
```

Comment:
I'm not sure it should be ignored. Judging by "Symbolizer was blocked from starting itself!" it seems a few sandbox features should be relaxed in `systemd-homed` to make it compatible with ASan

Reply:
Just to clarify, I was talking about "/lib/libLLVM"

Reply:
On second thought we're not interested in that memory leak either way so it should probably be ignored :-) I think what I was trying to say is that it's just a symptom of some sandbox setting being too strict probably

Reply:
Yeah, I wasn't completely sure where exactly it comes from, but I can't reproduce it with gcc (well, obviously, but I mean something similar), and it disappears when there are no leaks.

Nevertheless, I'll look into tweaking the sandbox restrictions as well, in case there's something else hiding.